app/assets/stylesheets/desktop/category-list.scss

```diff
@@ -109,6 +109,14 @@
     .latest {
       padding: 0 0 10px 10px;
     }
+
+    .blue-border {
+      border-color: #0088cc;
```

Comment:
Here, we should be using the colors in the scss variables. That way, if the admins change their theme, the colors will change as well.

Reply:
i have fixed it now @ZogStriP